### PR TITLE
Turn JDBC id and password into optional parameters

### PR DESCRIPTION
**Pull request by dennis-behm**
This is implementing #248 to allow the Common Backend Scripts to be used with the DBB FileMetadatastore.

### CODE REVIEW

File: Templates/Common-Backend-Scripts/dbbBuild.sh [hunk 1]
```diff
@@ -370,15 +370,11 @@ validateOptions() {
   esac
 
   if [ -z "${dbbMetadataStoreJdbcId}" ]; then
-    rc=8
-    ERRMSG=$PGM": [ERROR] DBB database user is required. rc="$rc
-    echo $ERRMSG
+    echo $PGM": [WARNING] Db2 JDBC User not set. The usage of the Db2 DBB Metadatastore is recommended."
```
Comment (M-DLB) [inline Templates/Common-Backend-Scripts/dbbBuild.sh:373]: Could we rephrase the last sentence to:
For the DBB metadata database, the use of Db2 (z/OS or LUW) is recommended.

File: Templates/Common-Backend-Scripts/dbbBuild.sh [hunk 2]
```diff
@@ -370,15 +370,11 @@ validateOptions() {
...
   fi
 
   if [ -z "${dbbMetadataStoreJdbcPwdFile}" ]; then
-    rc=8
-    ERRMSG=$PGM": [ERROR] DBB database password file is required. rc="$rc
-    echo $ERRMSG
+    echo $PGM": [WARNING] Db2 JDBC Password file not set. The usage of the Db2 DBB Metadatastore is recommended."
```
Comment (M-DLB) [inline Templates/Common-Backend-Scripts/dbbBuild.sh:377]: Could we rephrase the last sentence to:
For the DBB metadata database, the use of Db2 (z/OS or LUW) is recommended.

**Review by M-DLB — COMMENTED** [review]
Just a small change for clarity.

**Comment by dennis-behm** [comment]
Hi @M-DLB - can you please have a look to the new proposal for the warning message? https://github.com/IBM/dbb/pull/249/commits/dad6686e9dccb97bf8b5b647ac2bf04ed0b1288f

**Review by M-DLB — APPROVED** [review]
Approved, thank you @dennis-behm !

**Pull request merged**